CHANGELOG/CHANGELOG-2.6.md

```diff
@@ -0,0 +1,22 @@
+# Release notes for v2.6.0
+
+[Documentation](https://kubernetes-csi.github.io/docs/)
+
+# Changelog since v2.5.0
+
+## Changes by Kind
+
+### Feature
+
+- Add support to build arm/v7 images ([#139](https://github.com/kubernetes-csi/livenessprobe/pull/139), [@mauriciopoppe](https://github.com/mauriciopoppe))
+
+## Dependencies
```

Comment:
Should we also update the dependencies in this repo?

Reply:
/hold
yes, holding until https://github.com/kubernetes-csi/livenessprobe/pull/141 is merged